modules/oc-adm-by-example-content.adoc

```diff
@@ -346,6 +350,96 @@ Display and filter node logs
 
 
 
+== oc adm ocp-certificates monitor-certificates
+Watch platform certificates.
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Watch platform certificates.
+  oc adm ocp-certificates monitor-certificates
```

Comment:
the command failed with panic , have raised issue 
https://issues.redhat.com/browse/OCPBUGS-21612

Reply:
I'll leave this one in, since it isn't a problem with the command itself